addons/product/migrations/9.0.1.2/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,183 @@
+---Fields in module 'product'---
+product      / product.attribute        / attribute_line_ids (one2many) : NEW relation: product.attribute.line
+# New link to counterpart of existing many2many relation
+product      / product.attribute        / sequence (integer)            : NEW 
+# New field ---> Nothing to do
+
+product      / product.packaging        / code (char)                   : DEL
+product      / product.packaging        / ean (char)                    : DEL 
+product      / product.packaging        / name (text)                   : now required
+product      / product.packaging        / name (text)                   : type is now 'char' ('text')
+product      / product.packaging        / rows (integer)                : DEL required: required, req_default: 3
+product      / product.packaging        / ul (many2one)                 : DEL relation: product.ul, required: required, req_default: function
+product      / product.packaging        / ul_container (many2one)       : DEL relation: product.ul
+# TODO: remove NOT NULL constraint on these obsolete fields
```

Comment:
@StefanRijnhart Can you help me to clarify whether a specific action needs to take place to remove NOT NULL constraint on these fields? I have not seen any prior example in OpenUpgrate where a sentence such as ALTER TABLE table ALTER COLUMN column DROP NOT NULL would be used.